docusaurus/docs/e2e-test-a-plugin/getting-started.md

```diff
@@ -0,0 +1,148 @@
+
+### Installing Playwright
+
+Please refer to the [Playwright documentation](https://playwright.dev/docs/intro#installing-playwright) for instruction on how to install. `@grafana/plugin-e2e` extends Playwright APIs, so you need to have `Playwright/test` with a minimum version of 0.40.0 installed as a dev dependency in the package.json file of your plugin.
+
+## Set up `plugin-e2e`
+
+### Step 1: Installing @grafana/plugin-e2e
+
+Now open the terminal and run the following command in your plugin's project directory:
+
+<CodeSnippets
+snippets={[
+{ component: ScaffoldPluginE2InstallNPM, label: 'npm' },
+{ component: ScaffoldPluginE2EInstallYarn, label: 'yarn' },
+{ component: ScaffoldPluginE2EInstallPNPM, label: 'pnpm' }
+]}
+groupId="package-manager"
+queryString="current-package-manager"
+/>
+
+### Step 2: Configure Playwright
+
+Open the Playwright config file that was generated when Playwright was installed.
+
+1. Uncomment `baseUrl` and change it to `'http://localhost:3000'`.
+
+```ts title="playwright.config.ts"
+  baseURL: 'http://localhost:3000',
+```
+
+2. Playwright uses [projects](https://playwright.dev/docs/test-projects) to logically group tests that have the same configuration. We're going to add two projects. The first project called `auth` is a setup project will login to Grafana and store the authenticated state on disk. The second project `run-tests` runs all the tests in a browser of choice. By adding a dependency to the `auth` project we ensure login only happens once, and all tests in the `run-tests` project will start already authenticated.
+
+   Your Playwright config should have the following project configuration:
+
+<CodeSnippets
+snippets={[
+{ component: ScaffoldPluginE2EProjectsConfigNPM, label: 'npm' },
+{ component: ScaffoldPluginE2EProjectsConfigYarn, label: 'yarn' },
+{ component: ScaffoldPluginE2EProjectsConfigPNPM, label: 'pnpm' }
+]}
+groupId="package-manager"
+queryString="current-package-manager"
+/>
+
+### Step 3: Start Grafana
+
+Next, start up the latest version of Grafana on your local machine.
+
+<CodeSnippets
+snippets={[
+{ component: ScaffoldPluginE2EStartGrafanaNPM, label: 'npm' },
+{ component: ScaffoldPluginE2EStartGrafanaYarn, label: 'yarn' },
+{ component: ScaffoldPluginE2EStartGrafanaPNPM, label: 'pnpm' }
+]}
+groupId="package-manager"
+queryString="current-package-manager"
+/>
+
+If you want to start a specific version of Grafana, you can do that by specifying the `GRAFANA_VERSION` environment variable.
+
+```bash
+GRAFANA_VERSION=10.1.6 npm run server
+```
+
+### Step 4: Write tests
+
+In this example, we're using the panel edit page to test the query editor for a backend data source plugin. The plugin was scaffolded with the [create-plugin](https://github.com/grafana/plugin-tools/tree/main/packages/create-plugin) tool, and for this data source the query endpoint returns hard coded data points. This test asserts that the values `1` and `3` are being displayed in the `Table` panel.
```

Comment:
```suggestion
You are now ready to write your tests. In this example, we're using the panel edit page to test the query editor for a backend data source plugin. The plugin was scaffolded with the [create-plugin](https://github.com/grafana/plugin-tools/tree/main/packages/create-plugin) tool, and for this data source the query endpoint returns hard coded data points. This test asserts that the values `1` and `3` are being displayed in the `Table` panel.
```

Reply:
we could also optionally run the login/auth test prior to this to verify everything is working as expected with the setup?